resources/install-letsencrypt-cert.sh

```diff
@@ -49,6 +49,17 @@ elif [ "$OPENRESTY_INSTALL_CHECK" = "installed" ] || [ "$OPENRESTY_INSTALL_CHECK
     RELOAD_CMD="systemctl force-reload openresty.service"
 elif [ "$APACHE_INSTALL_CHECK" = "installed" ] || [ "$APACHE_INSTALL_CHECK" = "unpacked" ] ; then
     RELOAD_CMD="systemctl force-reload apache2.service"
+
+    if [[ "$(hostname)" == "$DOMAIN" ]]; then
+        DEFAULT_SITE_CONF="/etc/apache2/sites-enabled/000-default.conf"
+        if [[ -f "$DEFAULT_SITE_CONF" ]]; then # Checks if the default site is enabled
+            echo "Default Apache site may conflict with Let's Encrypt process"
+            a2dissite 000-default.conf && ${RELOAD_CMD}
```

Comment:
I don't think the jitsi-meet package should be touching that configuration, people maybe using it for some other stuff. 
If you think that maybe a problem we should print some warning to inform the user, so the user can decide what to do.